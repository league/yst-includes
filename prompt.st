
Highlight prompts and shell commands.

~~~~ {prompt="liucs:.*\\\$"}
liucs:~/cs643\$ cd a4/build
liucs:~/cs643/a4/build\$ make
[...]
32 bytes (32 B) copied, 0.000254176 s, 126 kB/s
cat vdi-header-2 >>vdi-header
cat vdi-header diskc.img >diskc.vdi
cp diskc.vdi /mnt/host
rm libc/entry.o
liucs:~/cs643/a4/build\$
~~~~
