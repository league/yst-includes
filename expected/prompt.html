<style>
.prompt { color: green; }
.cmd { color: blue; }
</style>

<p>Highlight prompts and shell commands.</p>
<pre class="sourceCode"><span class="prompt">liucs:~/cs643$</span><span class="cmd"> cd a4/build</span>
<span class="prompt">liucs:~/cs643/a4/build$</span><span class="cmd"> make</span>
[...]
32 bytes (32 B) copied, 0.000254176 s, 126 kB/s
cat vdi-header-2 &gt;&gt;vdi-header
cat vdi-header diskc.img &gt;diskc.vdi
cp diskc.vdi /mnt/host
rm libc/entry.o
<span class="prompt">liucs:~/cs643/a4/build$</span>
</pre>

